pkg/config/env.go

```diff
@@ -72,8 +72,9 @@ var Config = struct {
 	CORSEnabled bool `env:"FLAGR_CORS_ENABLED" envDefault:"true"`
 
 	// SentryEnabled - enable Sentry and Sentry DSN
-	SentryEnabled bool   `env:"FLAGR_SENTRY_ENABLED" envDefault:"false"`
-	SentryDSN     string `env:"FLAGR_SENTRY_DSN" envDefault:""`
+	SentryEnabled     bool   `env:"FLAGR_SENTRY_ENABLED" envDefault:"false"`
+	SentryDSN         string `env:"FLAGR_SENTRY_DSN" envDefault:""`
```

Comment:
Why do we prefix these env vars with `FLAGR_`? They seem pretty standard.

Reply:
It's optional, and in case flagr was deploying together with other apps in the same container, it's also good to isolate/prefix the environment variables just for the open source app.

People can definitely set the official `SENTRY_ENVIRONMENT` and leave `FLAGR_SENTRY_ENVIRONMENT` empty if they already have that configured. That will be respected by the official raven/sentry package.